src/sentry/static/sentry/app/views/projectGeneralSettings.jsx

```diff
@@ -19,19 +18,15 @@ import TextBlock from './settings/components/text/textBlock';
 import TextField from './settings/components/forms/textField';
 import {fields} from '../data/forms/projectGeneralSettings';
 
-const noMargin = {marginBottom: 0};
-
 const AutoResolveFooter = () => (
-  <Box p={2} pb={0}>
-    <PanelAlert type="warning" icon="icon-circle-exclamation" css={noMargin}>
-      <strong>
-        {t(
-          'Note: Enabling auto resolve will immediately resolve anything that has ' +
-            'not been seen within this period of time. There is no undo!'
-        )}
-      </strong>
-    </PanelAlert>
-  </Box>
+  <PanelAlert type="warning">
+    <strong>
+      {t(
+        'Note: Enabling auto resolve will immediately resolve anything that has ' +
+          'not been seen within this period of time. There is no undo!'
```

Comment:
>There is no undo! 

is a little bit harsh imo, but i know this is outside the scope of your PR

Reply:
I think it's supposed to be harsh